src/rules/folio-number-rules.ts

```diff
@@ -1,3 +1,3 @@
 export const FolioNumberRules: Array<Function> = [
-  v => (!v || v.length <= 50) || 'Cannot exceed 50 characters' // maximum character count
+  v => (!v || v.trim().length <= 50) || 'Cannot exceed 50 characters' // maximum character count
```

Comment:
This is needed because, apparently, the v-text-field component evaluates the rules on the raw value, not the trimmed v-model variable.